### PR TITLE
Dynamically fetch the latest release of NetBird on startup

### PR DESCRIPTION
**Pull request by braginini**

### CODE REVIEW

**Review by mlsmaycon — APPROVED** [review]

File: src/components/addpeer/WindowsTab.tsx [hunk 1]
```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
 import { Button } from "antd";
 import TabSteps from "./TabSteps";
 import { StepCommand } from "./types"
+import {getConfig} from "../../config";
+const {latestVersion} = getConfig();
```
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:7]: Codacy has a fix for the issue: [Replace `latestVersion` with `·latestVersion·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
const { latestVersion } = getConfig();
```

File: src/components/addpeer/WindowsTab.tsx [hunk 2]
```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
 import { Button } from "antd";
 import TabSteps from "./TabSteps";
 import { StepCommand } from "./types"
+import {getConfig} from "../../config";
```
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:6]: Codacy has a fix for the issue: [Replace `getConfig` with `·getConfig·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
import { getConfig } from "../../config";
```

File: src/components/addpeer/WindowsTab.tsx [hunk 3]
```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
...
+const {latestVersion} = getConfig();
 
 export const WindowsTab = () => {
 
-    const releaseVersion = '0.6.2'
+    const releaseVersion = latestVersion ? latestVersion : "v0.6.3"
```
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:11]: Codacy has a fix for the issue: [Replace `··const·releaseVersion·=·latestVersion·?·latestVersion·:·"v0.6.3"` with `const·releaseVersion·=·latestVersion·?·latestVersion·:·"v0.6.3";`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
  const releaseVersion = latestVersion ? latestVersion : "v0.6.3";
```

File: src/components/addpeer/WindowsTab.tsx [hunk 4]
```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
...
     const [steps, setSteps] = useState([
         {
             key: 1,
             title: 'Download and run Windows installer:',
             commands: (
-                <Button type="primary" href={`https://github.com/netbirdio/netbird/releases/download/v${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`} target="_blank">Download NetBird</Button>
+                <Button type="primary" href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`} target="_blank">Download NetBird</Button>
```
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:17]: Codacy has a fix for the issue: [Replace `········<Button·type="primary"·href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`}·target="_blank">Download·NetBird` with `<Button⏎··········type="primary"⏎··········href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`}⏎··········target="_blank"⏎········>⏎··········Download·NetBird⏎········`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
        <Button
```

Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:17]: Codacy has a fix for the issue: [Replace `········<Button·type="primary"·href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`}·target="_blank">Download·NetBird` with `<Button⏎··········type="primary"⏎··········href={`https://github.com/netbirdio/netbird/releases/download/${releaseVersion}/netbird_installer_${releaseVersion}_windows_amd64.exe`}⏎··········target="_blank"⏎········>⏎··········Download·NetBird⏎········`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
        <Button
```

File: src/components/addpeer/WindowsTab.tsx (hunk 2 above)
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:6]: Codacy has a fix for the issue: [Replace `getConfig` with `·getConfig·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
import { getConfig } from "../../config";
```

File: src/components/addpeer/WindowsTab.tsx (hunk 3 above)
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:11]: Codacy has a fix for the issue: [Replace `··const·releaseVersion·=·latestVersion·?·latestVersion·:·"v0.6.3"` with `const·releaseVersion·=·latestVersion·?·latestVersion·:·"v0.6.3";`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
  const releaseVersion = latestVersion ? latestVersion : "v0.6.3";
```

File: src/components/addpeer/WindowsTab.tsx (hunk 1 above)
Comment (braginini) [inline src/components/addpeer/WindowsTab.tsx:7]: Codacy has a fix for the issue: [Replace `latestVersion` with `·latestVersion·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
const { latestVersion } = getConfig();
```

**Pull request merged**